app/data/javafxdoc.json

```diff
@@ -1,8 +1,8 @@
 {
-  "current_release":                                  "19",
+  "current_release":                                "21.0.1",
+  "release_uuid":                                     "e5ab43c6aed54893b0840c1f2dcfca4d",
 
-  "javafx-documentation":                             "https://openjfx.io/javadoc/@@CURRENT_RELEASE@@/",
-  "javafxdoc_root":                                   "https://openjfx.io/javadoc/@@CURRENT_RELEASE@@/",
+  "javafxdoc_root":                                   "https://download.java.net/java/GA/javafx@@CURRENT_RELEASE@@/@@RELEASE_UUID@@/docs/api/",
```

Comment:
This seems to break the JavaFX Javadocs for me (locally via Docker). For example, when clicking the `Stage` link in `JavaFX Application Basic Structure By Example` (`/learn/javafx/structure/`), it forwards me to https://download.java.net/java/GA/javafx22/@@RELEASE_UUID@@/docs/api/javafx.graphics/javafx/stage/Stage.html which is an invalid link.